apps/overlay/src/views/lowerthirds/LowerTicker.o.tsx

```diff
@@ -1,15 +1,36 @@
 import React from "react"
 import { useLt } from "utils/hooks"
-import { Box, Text } from "@mantine/core"
+import { Box, Group, Stack, Text } from "@mantine/core"
+import useRoom from "../../hooks/useRoom.hook"
 
 const LowerTicker = () => {
+  useRoom()
   const { ticker } = useLt()
   return (
-    <Box>
-      <Text>HELLO PLS HELP</Text>
-      <Text>{ticker.headline.text}</Text>
-      <Text>{ticker.scrollerText.text}</Text>
-    </Box>
+    <Group align={"center"}>
+      <Stack justify={"center"} spacing={"xs"} align={"center"}>
```

Comment:
Optional clean up job: If the properties are string, no need to encapsulate then with `{}`